docs/support.md

```diff
@@ -29,6 +29,9 @@ Alternatively, you can use the following third party entities for their paid ser
 
 &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; We have been using Caprover in many project including government server as well as private servers with single and miltiple instances for large scaled applications (with more than 500k traffic) to small scaled. Mainly we used caprover for hosting dockerized PHP applicationa and other reactjs and vuejs based static web applications. If you need any help, you can contact us: [yourfriends@integrid.solutions](mailto:yourfriends@integrid.solutions)
 
+##### [PremoWeb Internet Services](https://premoweb.com)
+&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; Got a complicated application stack or many applications you'd like to move to Caprover? PremoWeb is a United States based business now offering application migration services for those who want to make the migration. Schedule a free consulation with owner Nick Maietta at 888-PREMOWEB (888-773-6693) or via email at `nick at/premoweb/dot/com`. You can also find Nick on the [CapRover Slack Group](https://join.slack.com/t/caprover/shared_invite/zt-o8obmx7o-x9icEG5fDAtvE9zhHlltzQ).
+
```

Comment:
Nice! 

You can remove the link, it's already in the footer of the site. Duplicating links make it more error prone when we need to update the invite link (it happens every now and then when the previous link gets abused by bots).

Reply:
I've removed the link in the last commit, but don't see that commit here. Marking is resolved for the moment.